v2/pkg/capture_group_extractors/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package extractors implements extractors for http response
+// data retrieval.
+package capture_group_extractors
```

Comment:
Names with underscore aren't standard go practices, i suggest you change the name of the extractor to something without underscores. Also, if this could be an extension of the regex extractor itself without being a separate package, it would be better!